muzzle/src/main/java/io/opentelemetry/javaagent/tooling/HelperInjector.java

```diff
@@ -52,6 +52,10 @@ public class HelperInjector implements Transformer {
   private static final TransformSafeLogger logger =
       TransformSafeLogger.getLogger(HelperInjector.class);
 
+  // a hook for static instrumentation used to save additional classes created by the agent
+  // see https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/static-instrumenter
+  public static StaticInstrumenterHook staticInstrumenterHook;
```

Comment:
Also, WDYT about making it private, and adding a static setter method? That's our usual pattern for global variables.

Reply:
Sure!